themes/docsy/assets/scss/section-index.scss

```diff
@@ -1,18 +1,19 @@
-// .section-index {
+.section-index {
 
-//     .entry {
-//         padding: .75rem;
-//     }
+    .entry {
+        padding: .75rem;
+    }
 
-//     h5 {
-//         margin-bottom: 0;
+    h5 {
```

Comment:
Similar comment as below. We shouldn't be making changes to the `themes` submodule.